### ISSUE TITLE
Move all the finished datasets to "Projects - Finished" in GDrive

### ISSUE THREAD

**Issue by ESapenaVentura** [ebi-ait/hca-ebi-wrangler-central#120]
**Description of the task:**

The datasets that are already finished* should be moved to the `PROJECTS - FINISHED` folder within GDrive, in order to avoid confusion about which datasets are finished and which are still in progress.

**Finished**: No more wrangling required, the dataset and its files are already valid, in ingest and gone through secondary review.

- [x] Assess how many folders there are in `PROJECTS - IN PROGRESS`
- [x] Assess how many of them are already finished
- [ ] Move the folders to `PROJECTS - FINISHED`

**Acceptance criteria for the task:**

- [ ] All the DCP2 datasets for the MVP have been moved to `PROJECTS - FINISHED`
- [ ] The list has been reviewed to identify potentially overlooked DCP1 datasets

**Comment by ESapenaVentura** [comment]
There are currently 75 folders under `IN PROGRESS`. I have created a list (Attached [here](https://github.com/ebi-ait/hca-ebi-wrangler-central/files/5130239/dataset_classification.rtf.gz)) that divides the folders in 4 categories:

- `IN PROGRESS`: These folders will be left in the same folder
- `DCP1 Datasets`: These will be moved to `PROJECTS - FINISHED`
- `DCP2 MVP/Not in DCP1`: These will be moved to `PROJECTS - FINISHED`
- `NOT SUPPORTED/ARCHIVED`: These will be moved to `PROJECTS - ARCHIVED`

**Comment by ESapenaVentura** [comment]
On the meeting @rays22 said he will take this task so re-assigning

**Comment by lauraclarke** [comment]
@rays22 can you articulate (short google doc is probably a good idea) your new proposal to allow us to easily find metadata spreadsheets/notes about an existing/finished project and understand what status a project has reached

Then we can move forward with the new proposal and close this ticket

Some important matters to discover
1. What is the goal of the new process, what will it allow wranglers to do?
2. How do we know is this process is successful in its goals?
3. When do we assess if the new process has worked?
4. What input do you need to be able to implement the new process

**Comment by lauraclarke** [comment]
@rays22 @ESapenaVentura I feel like this ticket has been superseded by all the data tracking updates

Is there any maintenance of the gdrive needed? what is it?

**Comment by rays22** [comment]
The gdrive contains subfolders for versions of metadata spreadsheets and other data (e.g. pdf paper reprints, data graph images etc.) associated with individual contributor submissions or datasets from archives. I am not sure if any maintenance of the gdrive is needed, at least not by moving folders around.

* [PROJECTS - IN PROGRESS](https://drive.google.com/drive/folders/118kh4wiHmn4Oz9n1-WZueaxm-8XuCMkA)
* [PROJECTS - FINISHED](https://drive.google.com/drive/folders/1FNRVqlhSwwTKoynIHhq5gsILGyRqd6F9)
* [PROJECTS - ARCHIVED](https://drive.google.com/drive/folders/1lbCb4gW2VnMwFbj3QFklBeWQHKN3n1Mh)

As I understand, there are two goals for this task:
1. Track the status of the datasets that are associated with each subfolder. The tracking would work by moving the subfolders to appropriate subfolder as a tracking status.
2. Identify metadata subfolders that might have gone under the radar and pick them up for wrangling/follow up.

Instead of doing Step 1. above, I am proposing to leave the subfolders for datasets where they are for good. I am also proposing that we add two new columns to the [dataset tracking sheet](https://docs.google.com/spreadsheets/d/1rm5NZQjE-9rZ2YmK_HwjW-LgvFTTLs7Q6MzHbhPftRE/) with the gdrive folder name and the URL for the associated subfolder. Then the status of the datasets associated with each folder can be looked up in the appropriate row and column in the [dataset tracking sheet](https://docs.google.com/spreadsheets/d/1rm5NZQjE-9rZ2YmK_HwjW-LgvFTTLs7Q6MzHbhPftRE/). In future, I would suggest to keep all the metadata subfolders in a single folder. We can rename one of the existing `PROJECTS -` folder without a status reference or create a new folder for the new sub-folders that need to be created (and keep track the dataset status in the [tracking sheet](https://docs.google.com/spreadsheets/d/1rm5NZQjE-9rZ2YmK_HwjW-LgvFTTLs7Q6MzHbhPftRE/)).

Tracking the dataset status at [one place](https://docs.google.com/spreadsheets/d/1rm5NZQjE-9rZ2YmK_HwjW-LgvFTTLs7Q6MzHbhPftRE/) would save the regular effort of trying to dataset tracking at two places and it would ensure consistency in statuses.

When adding the 75 folders (plus checking if there are new folders not yet on the list) to the [dataset tracking sheet](https://docs.google.com/spreadsheets/d/1rm5NZQjE-9rZ2YmK_HwjW-LgvFTTLs7Q6MzHbhPftRE/) it will become evident which folder is an orphan. In those cases we can follow them up and add those datasets to the tracking sheet and generate a list for potentially overlooked DCP1 datasets.

The new acceptance criteria would be as follows:
1. [ ] The [dataset tracking sheet](https://docs.google.com/spreadsheets/d/1rm5NZQjE-9rZ2YmK_HwjW-LgvFTTLs7Q6MzHbhPftRE/) have two new columns for the gdrive folder names and URLs associated with each relevant dataset.
2. [ ] The location information of the gdrive folders have been added to the two new columns.
3. [ ] The list has been reviewed to identify potentially overlooked DCP1 datasets.

@lauraclarke and @ESapenaVentura : What do you think about my proposal?

**Comment by ESapenaVentura** [comment]
We do store the gdrive folder in the tracking tickets, so although it makes it easier to find (no need to click twice), I don't really see the usefulness of adding any URL/folder name column to the tracking sheet

Also, this makes me think: We used to track progress with the folder structure, since we are going to track progress with ZH + dataset tracking sheet, is this really necessary? Can't we just get them all under the same directory?

**Comment by lauraclarke** [comment]
I agree with @ESapenaVentura If the status is tracked in the ticket/data tracking sheet, also having it in gdrive seems unnecessary

Having one folder with all the sheets in seems reasonable.

There are some upsides in also having a document link in the tracking sheet as it reduces the number of clicks to get from the tracking sheet to the spreadsheet but I totally get not wanting to maintain that

**Comment by rays22** [comment]
Thank you @lauraclarke and @ESapenaVentura for your feedback.
It appears to me that we all agree on that we should not use the folder structure for tracking status.
I also see the point of not maintaining the folder links in the tracking sheet in addition to what we already do which is to store that information in the github tickets for the datasets. Two clicks feel easier to me than two copy and paste actions in a huge spreadsheet.

How about if I keep only the last one of the acceptance criteria and complete that task that way? Then the completion of this task would not depend on an update to the dataset tracking sheet. Optionally, I can move the metadata subfolders in `ARCHIVED` and `FINISHED` into a single folder (e.g. `IN PROGRESS` --> renamed to simply `PROJECTS` or something else), but I think that would require input from all wranglers.

- [ ]  Compare the list of metadata subfolders with the datasets in the tracking sheet and identify potentially overlooked DCP1 datasets.

**Comment by lauraclarke** [comment]
When you say DCP1 datasets, do you mean contributor conversations which started and weren't finished during DCP1 or datasets which actually made it to DCP1

**Comment by rays22** [comment]
> When you say DCP1 datasets, do you mean contributor conversations which started and weren't finished during DCP1 or datasets which actually made it to DCP1

I think the intention of the original task description referring to DCP1 by @ESapenaVentura was to identify and follow up datasets/ contributor conversations which started during DCP1 but have not been finished and we also might have lost track of them. @ESapenaVentura , could you kindly confirm or correct my interpretation?

**Comment by ESapenaVentura** [comment]
My original intention with this ticket was to be able to get a clear vision of which projects were finished and which not

Currently it's impossible to tell just by looking at google drive because there are many datasets that have not had their location updated (so, even if they're finished, they are still in progress)

Ideally that could highlight "forgotten" datasets(as you mentioned), but that's a consequence rather than the intention of this ticket

**Comment by lauraclarke** [comment]
Given the data tracker and github should give us this view I wonder if this is needed anymore then, if you guys want to define a new process for how you store things in google drive, great but I don't think this ticket is needed, 

Please close it if you agree

**Comment by rays22** [comment]
> Given the data tracker and github should give us this view I wonder if this is needed anymore then, if you guys want to define a new process for how you store things in google drive, great but I don't think this ticket is needed,
> 
> Please close it if you agree

I agree, because I trust that we have not lost sight of any important and unfinished dataset that we could uncover and unblock by reviewing the subfolders on the gdrive. @ESapenaVentura please, re-open the ticket if you disagree.
